homework-submission/my app/my-app/src/add.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class AddTodo extends React.Component {
+  state = {
+    description: '',
+    done: false
+  }
+  todo = (element) => {
+    this.setState({ [element.target.id]: element.target.value })
+  }
+  submit = (element) => {
+    element.preventDefault();
+    this.props.addTodo(this.state)
+    this.setState({
+      description: '',
+      done: false,
+    })
+  }
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.submit}>
+          <span>Add Todo</span>
+          <div>
+            <span>Todo description :</span>
+            <input type='text' id='description' onChange={this.todo}>
```

Comment:
What about `name` instead of `id` attribute?

names usually are for forms, `id` has another uses as a Css selector and many other things, so what if you need to have another id with the same id name afterwards?